packages/rx_bloc_cli/example/README.md

```diff
@@ -608,10 +608,10 @@ For details on the API contracts used in the Forgotten password feature, please
 [patrol_pub_lnk]: https://pub.dev/packages/patrol
 [patrol_cli_pub_lnk]: https://pub.dev/packages/patrol_cli
 [patrol_native_integration_lnk]: https://patrol.leancode.pl/getting-started#integrate-with-native-side
-[patrol_integration_test_lnk]: https://github.com/Prime-Holding/rx_bloc/blob/master/packages/rx_bloc_cli/example/docs/patrol_integration_test.md
```

Comment:
We have changed the strategy of using relative to full path multiple times. The goal is the have working links on pub.dev once we release the package. @DavidDWiser, please advise

Reply:
[Correct](https://github.com/Prime-Holding/rx_bloc/wiki/Guidelines-and-Code-of-Conduct). Since the readme in the example directory is used to display content on the pub.dev, if we didn't promote the relative to full path, we'd break the links.